README.md

```diff
@@ -125,9 +125,26 @@ public class MainActivity extends ReactActivity {
 
 ## iOS
 
-Add `Indy.framework` as dependency into your project. Beware that the library needs to be compiled with the same version of Swift as your project.
+1. Install CocoaPods dependencies:
+```
+pod install --project-directory=ios/
+```
+
+2. Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.frameworks` into that directory.
+
+3. Add `Indy.framework` as dependency into your project. Open `.xcworkspace` file in Xcode and in your project settings, tab General, section Frameworks, Libraries, and Embedded Content, click on plus. Then select Add Other -> Add files... and navigate to `Indy.framework` file on your disk. Beware that `Indy.framework` needs to be compiled with the same version of Swift as your project.
```

Comment:
It is important to note that the file is not added to the gitignore. Otherwise you will need to do this after every clone

Reply:
Also regarding the sentence:
"Beware that `Indy.framework` needs to be compiled with the same version of Swift as your project."

This is not true if "Indy.framework" is built with "Build Libraries for Distribution", which can be enabled by using:

`Select framework target -> Build Settings -> Build Libraries for Distribution (BUILD_LIBRARY_FOR_DISTRIBUTION) -> Yes`

See https://stackoverflow.com/a/63305234/10552895.

I think this is very useful to note